### ISSUE TITLE
Add Dockerfile or instructions

### ISSUE THREAD

**Issue by isleshocky77** [google/import-mailbox-to-gmail#45]
Any appetite for adding Dockerfile or at least documenting in the README?  I had conflicts while loading this in my current system so decided to do it through docker instead.

## Setup

```
$ mkdir -p /opt/import-mailbox-to-gmail
$ cd /opt/import-mailbox-to-gmail
$ git clone https://github.com/google/import-mailbox-to-gmail.git build
```

## Create Dockerfile

```
# ./build/Dockerfile

FROM python:2

WORKDIR /usr/src/app
COPY requirements.txt ./
RUN pip install --no-cache-dir -r requirements.txt

COPY import-mailbox-to-gmail.py .

ENTRYPOINT [ "python", "./import-mailbox-to-gmail.py" ]
```

## Build

**NOTE** I had to add `oauth2client` to `requirements.txt` for it to work properly

```
$ cd ..
$ docker build -t import-mbox-to-gmail build/
```

## Get Data

```
$ mkdir -p data/auth
$ cp ~/Downloads/mail-import-013546-135458687.json data/auth/ # Authentication file
$ mkdir -p data/mbox/user1@example.org
$ cp ~/Downloads/label-1.mbox data/mbox/user1@example.org/
```

## Run

```
$ docker run --rm -it -v "$PWD/data:/data" \
       import-mbox-to-gmail \
          --json /data/auth/mail-import-013546-135458687.json \
          --dir /data/mbox
```

**Comment by eesheesh** [comment]
Hi @isleshocky77, that's a great idea, thanks! A pull request will be welcome. :)

**Comment by isleshocky77** [comment]
Closed by #47
